PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please add a line in the relevant section of [CHANGELOG.md](https://github.com/ImperialCollegeLondon/virtual_rainforest/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Optimization (back-end change that speeds up the code)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+
+
+# Key checklist:
+
+- [ ] No style issues: `$ flake8`
```

Comment:
As we're using ``pre-commit`` hooks, I'd tell users to make sure they've run those (which includes a hook for ``flake8``) instead.
```suggestion
- [ ] Make sure you've run the `pre-commit` checks: `$ pre-commit run -a`
```